core/org.wso2.carbon.ui/src/main/java/org/wso2/carbon/ui/internal/CarbonUIServiceComponent.java

```diff
@@ -262,6 +255,9 @@ public void start(BundleContext context) throws Exception {
         //Registering filedownload servlet
         Servlet fileDownloadServlet = new ContextPathServletAdaptor(new FileDownloadServlet(
                 context, getConfigurationContextService()), "/filedownload");
+//        Dictionary<String, Object> fileDownloadServletProperties = new Hashtable<>();
```

Comment:
Will have to migrate the filedownload and fileupload services to Whiteboard pattern.